src/App.tsx

```diff
@@ -1,5 +1,11 @@
+import List from "./List.tsx";
+
 function App() {
-  return <>starting point</>;
+  return (
+    <>
```

Comment:
I noticed you're using a React fragment tag, but it's unnecessary in this case since you're already returning a single element. Could you please remove the fragment tag to simplify the code?